shared/ui/src/components/ListHeader/index.tsx

```diff
@@ -0,0 +1,91 @@
+/** @jsxImportSource @emotion/react */
+import styled from '@emotion/styled';
+import { Padding, FlexBox } from '../../layout';
+import { Text } from '../Text';
+import { TextPropsKey } from '../Text';
+
+
+export interface ListHeaderProps {
+    title:string,
+    description: string;
```

Comment:
description은 선택적으로 받는 값이니까 물음표를 붙여 주는 게 더 좋을 것 가타요
```suggestion
    description?: string;
```

Reply:
꺄 이건몰랐는데 꿀팁감사용 헤헤